rtfs.go

```diff
@@ -102,6 +102,24 @@ func (im *IpfsManager) Pin(hash string) error {
 	return nil
 }
 
+// PinUpdate is used to update one pin to another, while making sure all objects
+// in the new pin are local, followed by removing the old pin.
+//
+// This is an optimized version of pinning the new content, and then removing the
+// old content.
+//
+// returns the new pin path
+func (im *IpfsManager) PinUpdate(fromPath, toPath string) (string, error) {
```

Comment:
do you mean `PinMigrate` or `PinMove`?

Reply:
I would also make the naming simpler 
```suggestion
func (im *IpfsManager) PinUpdate(from, to string) (string, error) {
```

Reply:
No the IPFS command is `ipfs pin update` so I felt it best to stick with the naming

Reply:
ah I see